src/dosbox.cpp

```diff
@@ -1192,27 +1193,40 @@ void DOSBOX_Init()
 	// DOS locale settings
 
 	secprop->AddInitFunction(&DOS_Locale_Init, changeable_at_runtime);
+
 	pstring = secprop->Add_string("locale_period", when_idle, "modern");
 	pstring->Set_help(
-	        "Set locale epoch ('modern' by default). Historic settings (if available\n"
-	        "for the given country) try to mimic old DOS behaviour when displaying\n"
-	        "information such as dates, time, or numbers, modern ones follow current day\n"
-	        "practices for user experience more consistent with typical host systems.");
-	pstring->Set_values({"historic", "modern"});
+	        "Set locale epoch ('native' by default).\n"
+	        "  historic:  if data is available for the given country, tries to mimic old DOS\n"
+	        "             behavior when displaying time, dates, or numbers\n"
+	        "  modern:    follow current day practices for user experience more consistent\n"
```

Comment:
Individual setting descriptions should be full sentences (start with upper case, and period at the end). E.g.:

```
  historic:  If data is available for the given country, tries to mimic old DOS
	     behavior when displaying time, dates, or numbers."
```

Reply:
OK, changed.